cmd/trayscale/systray.go

```diff
@@ -2,26 +2,32 @@ package main
 
 import (
 	_ "embed"
+	"fmt"
 
+	"deedles.dev/trayscale/internal/tsutil"
 	"fyne.io/systray"
 )
 
 type tray struct {
-	showItem *systray.MenuItem
-	quitItem *systray.MenuItem
+	showItem     *systray.MenuItem
+	quitItem     *systray.MenuItem
+	nodeInfoItem *systray.MenuItem
```

Comment:
Nitpick: I'd rename this to something like `selfInfoItem` or `selfNodeItem` or something so that it's more obvious which node it's talking about.

Also, while it's not necessary now, it might be nice to copy the local IP into the clipboard when the item is clicked. Ideally, at some point, I'd like to add _all_ known nodes and their IPs to the menu and be able to copy any IP when clicked much like how the Windows client works.

Reply:
Renamed to `selfNodeItem`. 

As for displaying other nodes - it makes sense and would be a nice feature. We can definitely move towards listing IPs like:

Main menu:
* Self IPv4
* Self IPV4
* Submenu node X:
  * Node X IPv4
  * Node X IPv6
* Submenu node Y:
  * Node Y IPv4
  * Node Y IPv6

Or

Main menu:
* Self IPv4
* Self IPV4
* Submenu other nodes:
  * Submenu node X:
    * Node X IPv4
    * Node X IPv6
  * Submenu node Y
    * Node Y IPv4
    * Node Y IPv6

As for clipboard - seems nice, I have no knowledge on how to handle that yet and how it works with Wayland.

Reply:
Gtk takes care of the clipboard. Take a look at how the IP copying buttons work on the peer pages in `cmd/trayscale/app.go`.

Reply:
Yup, I've seen it after commenting. 

But this is not the way I would see the tray go.
1. Expecting a tray app (that coincidentally has a GTK4 frontend too) to use GTK to manage clipboard is bloody awful.
2. Somehow hooking into respective buttons seems impossible too
3. There might be a way to use `gdk.Clipboard` directly. But it's a part of wider GTK too. I guess that's a way out if there's nothing better in the ecosystem.